### DISCUSSION TITLE
Marketing proposal - Xposure Today

### DISCUSSION THREAD

**Discussion by XposureToday** [akash-network/community#478] [Governance Proposals]
Hi there! Hopefully, you are doing fine.

I am the head of marketing at Xposure Today. We were researching Akash Network and were wondering if we could offer any of our services. 

I will hereby kindly send more information about us.

You can find our cases on our website: https://www.xposure.today/cases 

At Xposure Today we always create a tailored-made campaign for our clients, service examples may include:

Full DeFi preparation
o	We provide a full turn-key solution for DeFi project start. (Audit optional)
STO/IDO/ICO/IEO Setup
o	We create the full turn-key solution from the essentials for your successful STO/ICO/IEO launch.
Marketing influencers (All tiers possible)
o	YouTube/Twitter influencers from all tiers.
Growth Hacking packaged (Reddit(Top crypto-related subreddits), 4chan, Bitcointalk, Quora and many more.
o	Engaging hot/trending topics about your project.
Search engine optimization (keyword research for target audience, engaging content)
o	Technical, Internal and External optimization to receive much more engagement.
Social media management (tailored made social media strategy, including gfx from our end)
o	Receive our professional guidance to create much more engaging content across social media platforms.
Community management (24/7 Telegram/Discord management - 3 shifts of 8.5h)
o	Add 3 professional moderators to your channel to create a much more mature group.
Top media PR (Please inquire for more info)
o	PR articles on tier1 websites such as: Yahoo, Bloomberg and many more.
o	PR articles for other websites (Packages on our website on the worldwidepr page)
Other crypto-related services: (DexTools trending/banner, Instagram/Facebook ad banners, Twitter ad-banners, BSCScan/Etherscan ad-banners, Reddit and 4chan ad banners.

At Xposure Today we never offer one particular service package as other agencies, we try to get in touch with our client to receive a higher level of the project's structure and understanding, hence we always create a package based on our clients’ wishes. 

What we exactly would propose depends on your interests. We have done our research and we think a little push would help a lot. We could offer a lot of services, you can find them above. We could for example schedule a call to discuss what you may be looking for, and after that, we will come up with a tailored plan according to your interests. 

Feel free to schedule a call with us: https://calendly.com/xposuretoday/30min 

Please take your time to review our proposal and feel free to ask any questions

Looking forward to hearing from you! 🙏🏼

**Socials**
Website: https://www.xposure.today/ 

![XposureServiceList-2](https://github.com/akash-network/community/assets/159130597/8d8745b5-46d4-488f-a22b-c66936199d7b)
[Xposure Today PR Packages.xlsx.pdf](https://github.com/akash-network/community/files/14178048/Xposure.Today.PR.Packages.xlsx.pdf)
[Xposure today.pdf](https://github.com/akash-network/community/files/14178050/Xposure.today.pdf)
[XposureServiceList.pdf](https://github.com/akash-network/community/files/14178053/XposureServiceList.pdf)
![XposureServiceList-1](https://github.com/akash-network/community/assets/159130597/02fa6c63-b9ef-4b55-9b79-f3ccdf7541a8)

**Comment by Linkan4242** [comment]
Guys , this guys are totally scam. Dont believe them. They ger money then disappear. They show the prices low so you can check with them but fully scam . Dont believe them

**Comment by ghost** [comment]
No, thank you

**Comment by rodri-r** [comment]
Akash core team, Insiders and Vanguards already manage all of this.  So no thank you!!!
